controllers/etcd_controller.go

```diff
@@ -224,9 +224,16 @@ func buildPredicate(ignoreOperationAnnotation bool) predicate.Predicate {
 }
 
 // +kubebuilder:rbac:groups=druid.gardener.cloud,resources=etcds,verbs=get;list;watch;create;update;patch;delete
-// +kubebuilder:rbac:groups=druid.gardener.cloud,resources=etcds/status,verbs=get;update;patch
+// +kubebuilder:rbac:groups=druid.gardener.cloud,resources=etcds/finalizers,verbs=get;update;patch;create
```

Comment:
@shreyas-s-rao can you explain what do we need this for? IIUC, there is no `finalizers` subresource for etcd.

Reply:
@timuthy this seems to be added by https://github.com/gardener/etcd-druid/pull/336, probably due to the introduction of [finalizer subresource](https://github.com/kubernetes/enhancements/pull/3286).